packages/wrangler/src/entry.ts

```diff
@@ -35,6 +39,8 @@ export async function getEntry(
         ? path.resolve(config.site?.["entry-point"])
         : // site.entry-point could be a directory
           path.resolve(config.site?.["entry-point"], "index.js");
+    } else if (args["experimental-public"]) {
+      file = path.resolve(__dirname, "../templates/no-op-worker.js");
```

Comment:
Wasn't immediately obvious that the facade imports this worker - on the surface it looks like we're 404ing all requests 😅